Classes/Common/Solr.php

```diff
@@ -324,12 +324,6 @@ public function search()
         $params['filterquery'] = isset($params['filterquery']) ? $params['filterquery'] : [];
         // Restrict the fields to the required ones.
         $params['fields'] = 'uid,id';
-        // Extend filter query to get all documents with the same uids.
```

Comment:
Actually, this breaks my changes and I cannot see any usecase for this code.

Reply:
If I remember correctly, this is necessary to get all hits within a document. Since we split a METS document into several parts (i. e. pages containing fulltext) which are indexed as separate Solr documents, we have to merge these parts again when searching.

Reply:
I think, I checked this and it is still working. But I will have another look at.

Reply:
@frank-ulrich-weber Do you remember why we added this join?